### PR TITLE
Update message.js

### PR DESCRIPTION
**Pull request by Ciach0**
Added a bit of code (prefix.toLowerCase()).
That line of code allows the user to write the prefix in every case (e.g. d! and D!). This is useful if you are on mobile and you do have an autocorrect in keyboard.

**Please describe the changes this PR makes and why it should be merged, as well as any and all proof of successful tests:**

As mentioned above, this makes the prefix possible to write in every case (the prefix is case insensitive).

**Semantic versioning classification:**  
- [x] This PR changes the framework's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to README, etc.

### CODE REVIEW

File: events/message.js [hunk 1]
```diff
@@ -29,13 +29,13 @@ module.exports = (client, message) => {
...
   const prefix = prefixes.find(p => message.content.startsWith(p));
-  if (message.content.indexOf(prefix) !== 0) return;
+  if (message.content.toLowerCase().indexOf(prefix.toLowerCase()) !== 0) return;
 
   // Here we separate our "command" name, and our "arguments" for the command.
   // e.g. if we have the message "+say Is this the real life?" , we'll get the following:
   // command = say
   // args = ["Is", "this", "the", "real", "life?"]
-  const args = message.content.slice(prefix.length).trim().split(/ +/g);
+  const args = message.content.slice(prefix.toLowerCase().length).trim().split(/ +/g);
```
Comment (kyranet) [inline events/message.js:38]: `prefix.toLowerCase().length` and `prefix.length` are **always** the same. Please revert this change.

File: events/message.js [hunk 2]
```diff
@@ -29,13 +29,13 @@ module.exports = (client, message) => {
   // the bots user id (a mention).
   const prefixes = [settings.prefix, `<@!${client.user.id}>`];
   const prefix = prefixes.find(p => message.content.startsWith(p));
-  if (message.content.indexOf(prefix) !== 0) return;
+  if (message.content.toLowerCase().indexOf(prefix.toLowerCase()) !== 0) return;
```
Comment (kyranet) [inline events/message.js:32]: This code is completely wrong, line 32 always evaluated to true, I have no idea why that existed, but the check should have been:

```javascript
if (!prefix) return;
```

Either way, the correct way to do this is by doing this instead:

```javascript
const prefixes = [settings.prefix.toLowerCase(), `<@!${client.user.id}>`];

const content = message.content.toLowerCase();
const prefix = prefixes.find(p => content.startsWith(p));
if (!prefix) return;
```

**Comment by YorkAARGH** [comment]
I'm not sure why you're adding `.toLowerCase()` like that, the code is perfectly fine and functional as it is, I won't close this PR _yet_ but please explain the motivation behind PR'ing this change.

**Review by kyranet — CHANGES_REQUESTED** [review]
If you tested this code, you'll have clearly seen that this code effectively does not work. You don't slap `.toLowerCase()` everywhere and expect it to work, the modifications were needed in only one place, and that's line 31 (modified line 30 as well to apply it to the configurable prefix).

**Comment by YorkAARGH** [comment]
> Be a dick, be treated like a dick

It was brought to my attention that you opened, and closed a PR to get the 100th PR.

Not on my watch.
